PendingReleaseNotes.md

```diff
@@ -17,6 +17,8 @@
 
 - Rook Standalone
   - Standalone mode has been disabled and is no longer supported. A Kubernetes environment is required to run Rook.
+- Rook-operator now deploys in `rook-system` namespace
```

Comment:
could you also add a note about the new volume plugin as a new feature? And mention how the prereqs are simpler and a link to the docs that describe the need to restart the kubelet in pre-1.7

Reply:
Yes, this is what was agreed upon. Keep in mind that this is the namespace we chose in the manifest in our `example` folder. Users can deploy this anywhere they want. But I will add a few sentences  stating this.